grid_helper.js

```diff
@@ -433,6 +434,10 @@ function buildView(divId, initialLayout, initialContent=[], numCols=12, gridRowH
           });
           that.locationHash = '#' + name;
           window.location.hash = that.locationHash;
+          that.currentTableau = name;
+        },
+        isCurrentTableau: function(name) {
+          return (this.currentTableau === name) ? true : false;
```

Comment:
Better and clearer this way:

```javascript
isCurrentTableau: function(name) {
  return this.currentTableau === name;
}
```

Pedantic Note: The variable `currentTableau` might be better named `currentTableauName`. The former implies an object, the latter is clearer. It's always helpful to make clear the distinction between an object and the types of things that point or refer to it. In this case, the Name is being used as a key and could be named `currentTableKey` or `currentTableauID`.